src/Symfony/Bundle/FrameworkBundle/Console/Application.php

```diff
@@ -70,12 +73,28 @@ public function doRun(InputInterface $input, OutputInterface $output)
 
         $this->setDispatcher($this->kernel->getContainer()->get('event_dispatcher'));
 
+        if ($this->registrationErrors) {
+            $this->renderRegistrationErrors($output);
+        }
+
         return parent::doRun($input, $output);
     }
 
     /**
      * {@inheritdoc}
      */
+    final protected function doRunCommand(Command $command, InputInterface $input, OutputInterface $output)
```

Comment:
making it final when overwriting the method is a BC break. People extending the class were allowed to overwrite the method previously